build-support/bin/external_tool/python_test.py

```diff
@@ -0,0 +1,126 @@
+# Copyright 2025 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+from pathlib import Path
+
+from external_tool.python import replace_class_variables
+
+CONTENT = """\
+from __future__ import annotations
+
+from pants.core.goals.resolves import ExportableTool
+from pants.core.util_rules.external_tool import TemplatedExternalTool
+from pants.engine.platform import Platform
+from pants.engine.unions import UnionRule
+from pants.option.option_types import ArgsListOption, SkipOption
+from pants.util.strutil import help_text
+
+
+class Cue(TemplatedExternalTool):
```

Comment:
I could imagine a real Cue backend in the future :) We have tended to use things like "Fortran" as a test domain, because that is an unlikely backend.

For now this is fine though!

Reply:
True, we already have a Cue backend, but this is not a problem because it's just text, it's not trying to import it, so there will be never a collision here